pkg/interactor/cloud.go

```diff
@@ -0,0 +1,41 @@
+package interactor
+
+import (
+	"errors"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/network/cloud"
+)
+
+var ErrTokenMissing = errors.New("no access token provided")
+
+// CloudInteractor ...
+type CloudInteractor struct {
+	cloud cloud.Cloud
+}
+
+// NewCloudInteractor ...
+func NewCloudInteractor(cloud cloud.Cloud) CloudInteractor {
+	return CloudInteractor{cloud}
+}
+
+// Authenticate ...
+func (ci *CloudInteractor) Authenticate(token string) error {
+	token, err := validateToken(token)
```

Comment:
too many `token` -- it's redundant :eyes: 
Also, bad practiceeee :sob: 
```suggestion
	t, err := validate(token)
```

Reply:
There was no need to return anything else other than the error itself. Done!